From 6d56ea47a4da4ab6a649ed6fccc1f7bc7e3026ad Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Tue, 17 Jan 2023 11:02:40 -0800
Subject: [PATCH] Also build dependencies

---
 libretro/Makefile.libretro | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libretro/Makefile.libretro b/libretro/Makefile.libretro
index 226b1910..6e42aa91 100644
--- a/libretro/Makefile.libretro
+++ b/libretro/Makefile.libretro
@@ -291,7 +291,7 @@ ifeq ($(STATIC_LIBCXX), 1)
 endif
 
 .PHONY: all
-all: $(TARGET)
+all: deps $(TARGET)
 
 .PHONY: deps
 deps:
-- 
2.34.1

