From b961637484a5f0463535a8c5f5d414cbac559d25 Mon Sep 17 00:00:00 2001
From: Garrett Brown <themagnificentmrb@gmail.com>
Date: Sat, 15 Apr 2023 10:52:49 -0700
Subject: [PATCH] Fix libretro print screen button

---
 libretro/src/libretro_dosbox.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libretro/src/libretro_dosbox.h b/libretro/src/libretro_dosbox.h
index ed82eb86..1c2e515c 100644
--- a/libretro/src/libretro_dosbox.h
+++ b/libretro/src/libretro_dosbox.h
@@ -111,7 +111,7 @@ constexpr std::tuple<retro_key, KBD_KEYS> retro_dosbox_map[]{
     {RETROK_PERIOD, KBD_period},
     {RETROK_COMMA, KBD_comma},
     {RETROK_SLASH, KBD_slash},
-    {RETROK_SYSREQ, KBD_printscreen},
+    {RETROK_PRINT, KBD_printscreen},
     {RETROK_PAUSE, KBD_pause},
     {RETROK_INSERT, KBD_insert},
     {RETROK_HOME, KBD_home},
-- 
2.37.0.windows.1

